### PR TITLE
Doc spelling fix - Part II

### PR DESCRIPTION
**Pull request by yenicapotediaz**
So.Many.Files

### CODE REVIEW

**Comment by guineveresaenger** [comment]
I approve of 7fa0b6e adding the punctuation back.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,32 +33,32 @@ Before requesting a new feature, please check open issues for any similar reques
...
+* Detailed steps to reproduce the bug
 * The version of kraken-lib you’re using
 * The expected behavior
 * The actual, incorrect behavior
 * Apply the label 'Bug'
 
 Search our [issue tracker](https://github.com/samsung-cnct/k2/issues) for existing issues (aka tickets) similar to yours. If you find one, please add your information to it as a comment.
 
-If you want to provide a patch with your bug report, please do by sending us a pull request (PR) as described in the section below. 
+If you want to provide a patch with your bug report, please do by sending us a pull request (PR) as described in the section below.
```
Comment (guineveresaenger) [inline CONTRIBUTING.md]: "please do so by sending" or "please send"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ Here we document how to use kraken-tools to work with a cluster.
 1. Generate latest configuration file, for example: `kraken generate`
 1. Create a Docker container that points to you (and mount your kraken-lib workspace) cluster config (generated earlier):
 `hack/dockerdev -c <PATH_TO_CONFIGS>/<YOUR_CONFIG>.yaml`
-1. In 15 to 45 seconds, you will be in the Bash terminal of the container 
+1. In 15 to 45 seconds, you will be in the Bash terminal of the container
 1. Bring up your cluster as you would normally when developing kraken-lib (including any flags you require):
```
Comment (guineveresaenger) [inline CONTRIBUTING.md]: This table has only 1's as numbers. It displays correctly though. Huh.

File: Documentation/kraken-configs/nodepool.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ We do not expect the same machine types to be used for each purpose, therefore e
 ### Root Options
 | Key Name        | Required     | Type    | Description|
 | ----------      | ----------   | ------- | --- |
-| name            | __Required__ | String  | node pool name |
+| name            | __Required__ | String  | Node pool name |
```
Comment (guineveresaenger) [inline Documentation/kraken-configs/nodepool.md]: Nodepool?

File: Documentation/security/README.md [hunk 4]
```diff
@@ -133,11 +133,11 @@ It should be accessible from the user side but also cluster side.
 The chart `openldap` is used for authentication backend of `dex`.
 
 ## Endpoints
-By default, it exposes ports 30443, 30080 for `dex`.
-User should use same URI which defined in config.yaml.
-And a port 31080 also exposed for phpLDAPadmin which included in `openldap`.
+By default, the chart exposes ports 30443, 30080 for `dex`.
+The user should use same URI which is defined in the config.yaml,
+and a port 31080 is also exposed for phpLDAPadmin which is included in `openldap`.
```
Comment (guineveresaenger) [inline Documentation/security/README.md]: "a" is redundant

File: Documentation/security/README.md [hunk 5]
```diff
@@ -487,12 +486,12 @@ tomcat-1609341292-8qdbm   0/1       ContainerCreating   0          7s
 ## ToDo
 #### Configuration
 1. Certificates
-To use dex for auth, passing dex's ca certificates is required.
+To use dex for auth, passing dex ca certificates is required.
 But there is size limitation using cloud-init with AWS.
 Now it just use same CA what kube apiserver uses.
```
Comment (guineveresaenger) [inline Documentation/security/README.md]: "uses the same CA as kube..."

File: Documentation/tags/README.md [hunk 6]
```diff
@@ -2,12 +2,12 @@
 
 ## Usage
 
-User can use certain part of ansible roles under /Kraken/roles directory through tags. To use tags with commands, user should set environment variable ( $KRAKEN_TAGS ) for the session bash executes those commands .
+User can use certain part of ansible roles under /Kraken/roles directory through tags. To use tags with commands, user should set environment variable ( $KRAKEN_TAGS ) for the session bash executes those commands.
```
Comment (guineveresaenger) [inline Documentation/tags/README.md]: "User can specify subset"?
"...for the session in which bash...."

File: Documentation/tags/README.md [hunk 7]
```diff
@@ -89,7 +89,7 @@ fabric_only,services_only
 ## List of tags and usage for ansible roles
 
 ### always
-**: The tag forces ansible to run a role that have 'always' tag. Use this for default role such as 'kraken.config'.**
+**: This tag forces ansible to run a role that have the 'always' tag. Use this for default role such as 'kraken.config'.**
```
Comment (guineveresaenger) [inline Documentation/tags/README.md]: "...that has the 'always' tag."

File: Documentation/tags/README.md [hunk 8]
```diff
@@ -89,7 +89,7 @@ fabric_only,services_only
...
 
 ### all
  **: If no tags are specified, this 'all' tag will be default tag.**
```
Comment (guineveresaenger) [inline Documentation/tags/README.md]: "...will be the default..."

File: CONTRIBUTING.md (hunk 2 above)
Comment (yenicapotediaz) [reply CONTRIBUTING.md]: The magic of Markdown. As long as they're numbers, it doesn't matter.

Comment (guineveresaenger) [reply CONTRIBUTING.md]: TIL...

**Pull request merged**